test/lib/schematics/schematic.option.spec.ts

```diff
@@ -38,19 +38,19 @@ describe('Schematic Option', () => {
       input: 'myApp',
       expected: 'my-app',
     },
-    {
+    /*{
       description: 'should allow underscore string option value name',
```

Comment:
I feel this could be a source of confusion: if I specify the `kebab` case, I would expect the convention to be respected.